environment.yml

```diff
@@ -0,0 +1,22 @@
+name: pyseldonenv
+channels:
+  - conda-forge
+
+dependencies:
+  - meson
+  - ninja
+  - python
+  - pybind11
+  - pytest
+  - meson-python
+  - fmt
+  - cmake
+  - cpp-argparse
+  - clang-format=18.1.1
+  - tomlplusplus
+  - catch2 
+  - pkg-config
+  # - pip
+  #   - sphinx
+  #   - sphinx-design
+  #   - sphinxcontrib-napoleon
```

Comment:
Good to remove these, all python dependencies (including `pybind11, pytest, meson-python`) should be handled by `pyproject.toml`